src/prompts/template.ts

```diff
@@ -61,18 +61,18 @@ export const templateMissingOptionPrompt = async (options, folderNameAnswers, de
     });
   }
 
-  let templateAnswers;
+  let templateAnswers:any;
 
   if (options.skipPrompts) {
     options = skipPromptsModified(options, defaultFolderName, notAmongTemplateCollection, defaultTemplate);
-    templateQuestions.template = defaultTemplate;
+      templateQuestions.template = defaultTemplate;
```

Comment:
Implement this solution here: https://github.com/code-collabo/node-mongo-cli/pull/228#discussion_r1525711197